revolv/administrator/views.py

```diff
@@ -12,15 +11,17 @@
 class AdministratorDashboardView(UserDataMixin, TemplateView):
     """Basic view for the Administrator dashboard. Shows the list of projects.
     """
-    template_name = 'administrator/dashboard.html'
+    template_name = 'base/dashboard.html'
 
     def get_context_data(self, **kwargs):
         context = super(AdministratorDashboardView, self).get_context_data(**kwargs)
-        context['proposed_projects'] = Project.objects.get_proposed()
-        context['active_projects'] = Project.objects.get_active()
-        context['completed_projects'] = Project.objects.get_completed()
-        context['all_projects'] = list(chain(context['proposed_projects'],
-                                             context['active_projects'], context['completed_projects']))
+        project_dict = {}
+        project_dict[('Proposed Projects', "proposed")] = Project.objects.get_proposed()
+        project_dict[('Active Projects', "active")] = Project.objects.get_active()
+        project_dict[('Completed Projects', "completed")] = Project.objects.get_completed()
```

Comment:
Using tuples is kind of hacky, could just have the display text as a key, and have some helper or dictionary to convert that to the corresponding css class.

It's fine to keep it this way, but add some comments!